src/hammer-vlsi/defaults.yml

```diff
@@ -312,6 +312,25 @@ vlsi.inputs:
     #  - y (int) - The Y coordinate of the bump assigned to this net
     assignments: []
 
+  # Pin placement mode. (str)
+  # Specifies how to arrange pins for your design.
+  # Valid options:
+  # none - This mode lets the CAD tool do whatever it wants (providing no constraints).
+  #   Typically this is sane but unpredictable.
+  # generated - Use the assignments list below
+  # auto - is unsupported but should look at hierarchy and guess where pins should go
+  pin_mode: none
+  pin:
```

Comment:
Should this be `vlsi.inputs.pin` or `vlsi.inputs.pins`? (modulo updating the `mode` above)

Reply:
This particular field is singular because the "assignments" are plural